### PR TITLE
ppc64le support

### PR DESCRIPTION
**Pull request by mkumatag**
This PR is to support ppc64le platform contains separate dockerfile and install script change.

### CODE REVIEW

File: containers/kubernetes/scripts/install-rpms.sh [hunk 1]
```diff
@@ -30,7 +30,12 @@ while [ $# -gt 0 ]; do
 done
 
 if [ -z "$COCKPIT_RPM_URL" ]; then
-    COCKPIT_RPM_URL="https://kojipkgs.fedoraproject.org/packages/cockpit"
+    if [ $arch == "ppc64le" ]
```
Comment (petervo) [inline containers/kubernetes/scripts/install-rpms.sh]: Shouldn't we just set COCKPIT_RPM_URL directly in the ppc64le docker file?

Comment (mkumatag) [inline containers/kubernetes/scripts/install-rpms.sh]: @stefwalter @petervo now updating COCKPIT_RPM_URL by default in the dockerfile, PTAL into latest commit.

**Comment by petervo** [comment]
@mkumatag, looks good to me, though I'm not setup to test it. Have you?

Is the plan to automatically build images from this anywhere? We update a copy with the latest version number with every release, here: https://github.com/cockpituous/cockpit/blob/master/containers/kubernetes/Dockerfile and that's where dockerhub is setup to build from. Do we need to do something similar for this?

**Comment by mkumatag** [comment]
@petervo I have done the testing with openshift origin registry-console and worked fine.

Yes, my next goal is to build this image and push into github. As this dockerfile is platform dependent and can be built only on ppc64le based system, so wondering what kind of automatic system in place to build this image in your environment?

**Comment by petervo** [comment]
We don't have anything at the moment that can build this automatically. But I'm ok merging this and figuring out building later. @stefwalter you ok with that?

**Comment by stefwalter** [comment]
Yup works for me. Running tests ...

**Pull request closed without merging**